src/traces/heatmap/hover.js

```diff
@@ -29,7 +29,6 @@ module.exports = function hoverPoints(pointData, xval, yval, hovermode, hoverLay
         zmask = cd0.zmask,
         range = [trace.zmin, trace.zmax],
         zhoverformat = trace.zhoverformat,
-        _separators = trace._separators,
```

Comment:
So we probably can :hocho: these now:

![image](https://user-images.githubusercontent.com/6675409/34051585-2d6e2686-e18d-11e7-9f97-9ed9884a53f0.png)

Reply:
:hocho: `trace._separators` in https://github.com/plotly/plotly.js/pull/2207/commits/592fc9eacd31f7d36a172edc6694ae570bca38bb